foundry.toml

```diff
@@ -2,4 +2,4 @@
 localhost="http://localhost:8545"
 # Remember to set a valid API_KEY and uncomment the rpc_endpoint definition
 # below to be able to use it:
-#goerli="https://eth-goerli.g.alchemy.com/v2/<API_KEY>"
+#sepolia="https://eth-sepolia.g.alchemy.com/v2/<API_KEY>"
```

Comment:
I don't think it's mentioned in the README that people should enter their API_KEY here to run things. Alternatively, the deploy procedure already uses the RPC_URL env var, wouldn't it be simpler if the `cast` commands also use that?

Reply:
Simplified things by using only `RPC_URL` at 306f5e1